payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/AddressElementTest.kt

```diff
@@ -572,6 +511,14 @@ class AddressElementTest {
     }
 }
 
+private suspend fun Flow<List<SectionFieldElement>>.usZipController(): TextFieldController {
+    return (first()[3] as RowElement).fields[1].controller as TextFieldController
```

Comment:
I think we should look these up by something other than their index. Can we use the country code?